kornia/losses/ssim3d.py

```diff
@@ -0,0 +1,105 @@
+import torch
+import torch.nn as nn
+
+import kornia.metrics as metrics
+
+
+def ssim3d_loss(
+    img1: torch.Tensor,
```

Comment:
```suggestion
    img1: Tensor,
```
`from kornia.core import Tensor`